topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -0,0 +1,629 @@
+---
+layout: tutorial_hands_on
+topic_name: transcriptomics
+tutorial_name: clipseq
+---
+
+# Introduction
+{:.no_toc}
+
+The eCLIP data provided here is a subset of the eCLIP data of RBFOX2 from a study published by *Nostrand et al.* (2016, [url](https://doi.org/10.1038/nmeth.3810)). The dataset contains the first biological replicate of RBFOX2 CLIP-seq and the input control experiment (fastq files). The data was changed and downsampled to reduce data processing time, consequently the data does not correspond to the original source pulled from *Nostrand et al.* (2016, [url](https://doi.org/10.1038/nmeth.3810)). Also included is a text file (.txt) encompassing the chromosome sizes of hg19 obtained from [UCSC](http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/hg19.chrom.sizes) and a genome annotation (.gtf) file taken from [Ensembl](http://ftp.ensemblorg.ebi.ac.uk/pub/release-74/gtf/homo_sapiens/).
```

Comment:
Can we have the year linked and remove the `url`?

Reply:
s/fastq/FASTQ/
I would also write `in FASTQ format`. You could even link this to wiki if you like.